src/dotnet-structuring.console/Program.cs

```diff
@@ -21,110 +21,114 @@ private void Setup(string[] args)
         {
             // Create a root command with some options
             var rootCommand = new RootCommand();
+            var teamplateOption = new Option("--template", "Choose a Template of the 'dotnet new' command")
+            {
+                Argument = new Argument<string>(getDefaultValue: () => "console")
+            };
+
+            var nameOption = new Option("--name", "Name of the Project being created")
+            {
+                Argument = new Argument<string>(getDefaultValue: () => "NewApp")
+            };
+
+            var outputOption = new Option("--output", "Output Directory")
+            {
+                Argument = new Argument<string>(getDefaultValue: () => @"C:\Develop")
+            };
 
+            var artifactsOption = new Option("-a", "Create artifacts Directory")
+            {
+                Argument = new Argument<bool>(getDefaultValue: () => false)
+            };
+            artifactsOption.AddAlias("--artifacts");
```

Comment:
can we add the short version as alias and the long version as primary option?

Reply:
++

Reply:
Seems like a good idea